Intersect.Client/Entities/Entity.cs

```diff
@@ -1386,6 +1386,99 @@ public virtual void DrawName(Color textColor, Color borderColor = null, Color ba
+
+                                break;
+                            case 0: // Attack when attacked.
+                                npcTagFile = Options.Npc.AttackWhenAttackedTagIcon;
+
+                                break;
+                            case 1: // Attack on sight.
+                                npcTagFile = Options.Npc.AttackOnSightTagIcon;
+
+                                break;
+                            case 3: // Guard.
+                                npcTagFile = Options.Npc.GuardTagIcon;
+
+                                break;
+                            case 2: // Neutral.
+                            default:
+                                npcTagFile = Options.Npc.NeutralTagIcon;
+
+                                break;
+                        }
+                        break;
+                }
+                // once we have the sprite file name we want to use, we can handle the tag texture to be drawn.
+                GameTexture npcTagTex = Globals.ContentManager.GetTexture(GameContentManager.TextureType.Tag, npcTagFile);
+                // before we draw the sprite, lets have it's position set.
+                switch (npcTagPos)
+                {
+                    case 0: // Position the tag above the Npc's name label.
+                    default:
+                        x = (int)Math.Ceiling(GetCenterPos().X) - npcTagTex.GetWidth() / 2;
+                        y = GetLabelLocation(LabelType.Name) - (npcNameSize.Y) - (npcTagTex.GetHeight() / 3);
+
+                        break;
+                    case 1: // Position the tag under the Npc's name label.
+                        x = (int)Math.Ceiling(GetCenterPos().X) - npcTagTex.GetWidth() / 2;
+                        y = GetLabelLocation(LabelType.Name) + (npcNameSize.Y) + 3;
+
+                        break;
+                    case 2: // Position the tag as prefix (left) of the Npc's name label.
+                        x = (int)Math.Ceiling(GetCenterPos().X) - (npcNameSize.X / 2) - (npcTagTex.GetWidth()) - 6;
+                        y = GetLabelLocation(LabelType.Name);
+
+                        break;
+                    case 3: // Position the tag as prefix (left) of the Npc's name label.
+                        x = (int)Math.Ceiling(GetCenterPos().X) + (npcNameSize.X / 2) + 6;
+                        y = GetLabelLocation(LabelType.Name);
+
+                        break;
+                }
+                // and finally, we draw the tag.
+                if (npcTagTex != null)
+                {
+                    Graphics.DrawGameTexture(npcTagTex, x, y, Intersect.Color.White);
+                }
+                if (npcTagTex == null)
+                {
+                    return;
+                }
+            }
+            if (!Options.Npc.ShowTags || this.GetType() == typeof(Player))
+            {
+                return;
+            }
```

Comment:
Not needed here.

Reply:
moved and changed to the beginning after the variables are declared
```
            if (!Options.Npc.ShowTags || this is Player)
            {
                return;
            }
```